bridge/src/index.ts

```diff
@@ -122,7 +122,8 @@ if (DEBUG !== undefined && DEBUG !== 'TRUE') {
     // It should be not able to run in mainnet because it is for test.
     if (DEBUG === 'TRUE' && CHAIN_ID !== 1) {
         nineChroniclesMonitor.subscribe(async event => {
-            console.log("Receipt", await minter.mint(event.sender, parseFloat(event.amount)));
+            const recipient = event.memo !== null ? event.memo : event.sender;
```

Comment:
IMO, if we can't get the recipient from bridge tx, a throwing error can prevent misremittance.

Reply:
I agree your opinion. I added the refund logic to resolve it in 5d5aa0e. Could you review it?